src/django/pfb_analysis/filters.py

```diff
@@ -59,4 +59,9 @@ def filter_latest(self, queryset, name, value):
 
     class Meta:
         model = AnalysisJob
-        fields = ['neighborhood', 'batch', 'status', 'latest']
+        fields = {'neighborhood': ['exact', 'in'],
+                  'neighborhood__name': ['exact', 'contains'],
+                  'neighborhood__label': ['exact', 'contains'],
+                  'batch': ['exact', 'in'],
+                  'status': ['exact', 'in'],
+                  'latest': ['exact', 'in']}
```

Comment:
Good catch. It's unfortunate that the directory field structure requires the fields to map to some filter type list. I tried using the `__all__` magic keyword to simply enable all filter types whether they made sense or not, but that broke things as it apparently doesn't work with cross-relationship fields included.

Added `in` as an option since I noticed the methods were using `__in` filters, so that seemed to be the current behavior, to do an inexact match. Do things still work as expected without a filter method type suffix? If so, I'll just leave them with just `exact`.